sql/sqlserver/server.go

```diff
@@ -94,10 +93,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Send the SQLRequest for SQL execution.
-	if httpStatus, err := s.execute(args, reply); err != nil {
-		http.Error(w, err.Error(), httpStatus)
-		return
-	}
+	s.Send(sqlwire.Call{Args: args, Reply: reply})
```

Comment:
I've made it Send because I intend to plumb through a Sender interface into your SQL execution code. Eventually once the client is talking to the SQLServer we can rename this.